geonode/settings.py

```diff
@@ -298,7 +298,8 @@
     'geonode.services',
 
     # QGIS Server Apps
-    'geonode.qgis_server',
+    # Only enable this if using QGIS Server
+    # 'geonode.qgis_server',
```

Comment:
I suggested we disabled this by default. GeoNode uses GeoServer by default and I personally never tested both apps enabled at the same time. So I disable this to avoid problems with GeoServer.

Reply:
By the way the package will still be tested in Travis on the QGIS Server matrix environment. The apps is included via local_settings.py.qgis.sample